.github/workflows/build_clio_docker_image.yml

```diff
@@ -38,11 +38,6 @@ on:
         description: Whether to strip clio binary
         default: true
 
-concurrency:
-  # Only cancel in-progress jobs or runs for the current workflow - matches against branch & tags
-  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Another workflow calls this as workflow_call, in which case `${{ github.workflow }}` value is taken from the caller, which makes workflow want to cancel itself and we get a deadlock issue.